test/json-ld.net.tests/Sorting/W3C/fromRdf-in.json

```diff
@@ -0,0 +1,58 @@
+﻿{
```

Comment:
This was the easiest way for me to create quads to provide the `FromRDF` method.